src/main/webapp/resources/js/userCalendar.js

```diff
@@ -232,12 +233,12 @@ $(function () {
         $('#bookingUpdatingDialog').dialog('close');
         var myDialog = $('#confirmation-dialog-div');
         myDialog.dialog('open');
+        $('#confirmYes').unbind('click');
         $('#confirmYes').click(function () {
             cancelBooking(info.id);
             myDialog.dialog('close');
         });
         $('#confirmNo').click(function () {
```

Comment:
i think here we too could use unbind, because it creates the same stack of functions when we press no button.